presto-main/src/main/java/com/facebook/presto/metadata/BuiltInTypeAndFunctionNamespaceManager.java

```diff
@@ -808,6 +810,8 @@ private List<? extends SqlFunction> getBuildInFunctions(FeaturesConfig featuresC
                 .scalar(ArrayNgramsFunction.class)
                 .scalar(ArrayAllMatchFunction.class)
                 .scalar(ArrayAnyMatchFunction.class)
+                .scalar(ArrayFindFunction.class)
+                .scalar(ArrayFindWithOffsetFunction.class)
```

Comment:
Hmm - you should not need two functions for this, I don't think. Just make it a variadic function and check in the constructor?

Reply:
Get an error when trying to combine the two classes to be one, there is the same error in PR https://github.com/prestodb/presto/pull/15170 too from the conversation.